src/main.js

```diff
@@ -1 +1,30 @@
-console.log('Hello, world!');
+// console.log('Hello, world!');
+// const total = document.querySelector('.form__total');
+// console.log(total);
+// // total.textContent = '';
+// const radio = document.querySelectorAll('input[type="radio"]');
+// console.log(radio);
+// // console.log(radio.checked === true);
+// function radioPayment() {
+//   if (radio.checked === true) {
+//     radio.textContent = '$300';
+//     return true;
+//   }
+// }
+// radio.forEach(radio => radio.addEventListener('change', radioPayment));
+// // radio.forEa.addEventListener('click', radioPayment());
+const amount = document.querySelector('.form__total span');
+console.log(amount);
+const inputs = Array.from(document.querySelectorAll('input[type="radio"]'));
+
+const annual = document.querySelector('input[value="Annually"]');
+const monthly = document.querySelector('input[value="Monthly"]');
+
+function changeCost() {
+  if (annual.checked === true) {
+    amount.textContent = '300';
+  } else {
+    amount.textContent = '30';
+  }
+}
+inputs.forEach(input => input.addEventListener('click', changeCost));
```

Comment:
Btw, `querySelectorAll()` returns a `NodeList` and you can directly run `forEach` on it without converting it into array (`Array.from(NodeList)`).

So this code:

```javascript
const inputs = Array.from(document.querySelectorAll('input[type="radio"]'));
inputs.forEach();
```

can be simpler:

```javascript
const inputs = document.querySelectorAll('input[type="radio"]');
inputs.forEach();
```

But, of course using the dot notation will be simplest:

```javascript
form.plan.forEach()
```